code/_core/datum/ability/voice_of_god.dm

```diff
@@ -0,0 +1,73 @@
+/ability/voice_of_god/harm
+	name = "Command: Harm"
+	desc = "Damages all enemies within 8 tiles, and makes them susceptable to further damage."
+	icon_state = "dash"
+
+	resource_type = MANA
+	var/damagetype/damage_type
+	var/atom/caller
+	var/word = "DIE"
+	category = "Voice Of God"
+
+/ability/voice_of_god/harm/proc/get_params(var/atom/victim)
+	. = list()
+	.[PARAM_ICON_X] = rand(0,32)
+	.[PARAM_ICON_Y] = rand(0,32)
+
+/ability/voice_of_god/harm/proc/do_damage(var/atom/victim,var/atom/caller)
+
+	if(is_living(victim) && is_living(caller) && caller != victim)
+		var/mob/living/L = victim
+		var/mob/living/L2 = caller
+		if(!allow_hostile_action(L2.loyalty_tag,L))
+			return FALSE
+	var/damagetype/DT = all_damage_types[damage_type]
+	var/list/params = get_params()
+	if(!victim.can_be_attacked(caller,src,params,DT))
+		return FALSE
+	var/atom/object_to_damage = victim.get_object_to_damage(caller,src,damage_type,params,TRUE,TRUE)
+	return DT.process_damage(caller,victim,src,object_to_damage,caller,1)
+
+/ability/voice_of_god/harm/on_cast(var/mob/caller,var/atom/target,var/atom/victim,location,params)
+
+	var/mob/living/L = caller
+	var/turf/T = get_turf(L)
+	play_sound('sound/effects/invoke_general.ogg',T, range_max = SOUND_RANGE)
+	L.do_say("<font color='#DD1C1F' size='4'>[word]</font>",FALSE)
+
+	for(var/mob/living/L2 in oview(T,8))
+		if(L2.dead)
+			continue
+		if(L2.loyalty_tag == L.loyalty_tag)
+			continue

```

Comment:
should replace this with !allow_hostile_action(L2.loyalty_tag,L)) and remove the one on line 22.

Reply:
Done and done. Should look better now.